subscription-manager.spec

```diff
@@ -10,9 +10,7 @@
 %global use_container_plugin 1
 %endif
 
-%if (0%{?rhel} || 0%{?fedora})
-%global dmidecode_version >= 3.12.2-2
-%endif
+%global dmidecode_arches %{ix86} x86_64
```

Comment:
This is correct, however I wanted to leave that enablement at a later stage, since it will require few more changes & additional testing (I created an internal card for this).